drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
> But what if the client gets http://www.example.com/api - which is already an entry point. This way you won't fetch any API documentation headers (I wouldn't expect those here).

Why not? If an API documentation exists, it should be referenced form any resources that is part of the API... and, if possible, also from every other resource the API publisher controls.

> From my experience APIs (http://www.example.com/api in this example) are usually added to the existing site (http://www.example.com/) and the main Url doesn't care about the API.

Yeah, that's the trickier case. It won't always be practical to do that but we should document that it is possible and enables automatic discovery.

> I wouldn't force the client to get the entry point via API documentation. I though both are independent to each other.

Of course not. It's important though to show that it is possible. The scenario I'm thinking about is that you for example browser a website. You get curious whether it would be possible to get data automatically. If the HTTP header is included, the browser (or an extension) can immediately tell you that an API is available without you having to look for it (on maybe a different subdomain).